GWAS/PCA.ipynb

```diff
@@ -1134,7 +1144,7 @@
     "    pca_final<-dat$pc_scores\n",
     "    \n",
     "    if (length(pops)>1) pca_final <- pca_final %>% filter(super_pop %in% pops| pop %in% pops) %>%\n",
-    "        mutate(${label_col if label_col else \"pop\"}=as.character(${label_col if label_col else \"pop\"}))\n",
+    "        mutate(${label_col if label_col is not \"\" else \"label_col_placeholder\"}=as.character(${label_col if label_col is not \"\" else \"label_col_placeholder\"}))\n",
```

Comment:
if you use "pop" as the placeholder you dont have to change this line? `if label_col` is effectively the same as `if label_col is not ""`. You dont have to do it otherwise.